tekton/publish.yaml

```diff
@@ -17,6 +17,9 @@ spec:
       description: TODO(tektoncd/pipeline#569) This is a hack to make it easy for folks to switch the registry being used by the many many image outputs
     - name: pathToProject
       description: The path to the folder in the go/src dir that contains the project, which is used by `ko` to name the resulting images
+    - name: releaseAsLatest
+      description: "Whether to tag and publish this release as Pipelines' latest"
```

Comment:
> as Pipelines' latest

shouldn't this be Triggers?

Reply:
🤦‍♂ indeed it should be!

Reply:
https://github.com/tektoncd/triggers/pull/524